integration/hurl/tests_ok/secret/secret_test.ps1

```diff
@@ -3,8 +3,8 @@ $ErrorActionPreference = 'Stop'
 
 hurl --test \
     --very-verbose \
-    --secret a=secret1 \
-    --secret b=secret2 \
+    --secrets-file tests_ok/secret/secrets0.env \
```

Comment:
This integration file should not be touched, we can make a new integration test specifi for --secrets-file in tests_ok/secret/secrets_file.ps1